modules/checking-mco-status.adoc

```diff
@@ -52,13 +70,55 @@ $ oc describe mcp worker
   Updated Machine Count:      3
 Events:                       <none>
 ----
++
+[NOTE]
+====
+If a node is being cordoned, the node is not included in the `Ready Machine Count`. It is included in the `Unavailable Machine Count`:
+
+.Example output
+[source,terminal]
+----
+...
+  Degraded Machine Count:     0
+  Machine Count:              3
+  Observed Generation:        2
+  Ready Machine Count:        2
+  Unavailable Machine Count:  1
+  Updated Machine Count:      3
+----
+====
+
+. To see each existing `MachineConfig` object:
++
+[source,terminal]
+----
+$ oc get machineconfigs
+----
++
+[source,terminal]
+----
+NAME                             GENERATEDBYCONTROLLER          IGNITIONVERSION  AGE
+00-master                        2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
+00-worker                        2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
+01-master-container-runtime      2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
+01-master-kubelet                2c9371fbb673b97a6fe8b1c52…     3.2.0            5h18m
+...
+rendered-master-dde...           2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
+rendered-worker-fde...           2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
+----
++
+Note that the `MachineConfig` objects listed as `rendered` are not meant to be changed or deleted. Expect them to be hidden at some point in the future. 
 
-. You can view the contents of a particular machine config (in this case, `01-master-kubelet`). The trimmed output from the following `oc describe` command shows that this `machineconfig` contains both configuration files (`cloud.conf` and `kubelet.conf`) and a systemd service
+. You can view the contents of a particular machine config (in this case, `01-master-kubelet`). The output from the following `oc describe` command shows that this `MachineConfig` object contains both configuration files (`cloud.conf` and `kubelet.conf`) and a systemd service
 (Kubernetes Kubelet):
 +
 [source,terminal]
 ----
 $ oc describe machineconfigs 01-master-kubelet
+----
++
+[source,terminal]
```

Comment:
You could add an ".Example output" line before this one.

Reply:
Outdated